csharp/ql/src/semmle/code/csharp/dataflow/internal/tainttracking1/TaintTrackingParameter.qll

```diff
@@ -0,0 +1,5 @@
+import semmle.code.csharp.dataflow.internal.TaintTrackingUtil as Public
+
+module Private {
+  import semmle.code.csharp.dataflow.DataFlow::DataFlow as DataFlow
+}
```

Comment:
I would prefer
```
import semmle.code.csharp.dataflow.internal.TaintTrackingPublic as Public

module Private {
  import semmle.code.csharp.dataflow.DataFlow::DataFlow as DataFlow
  import semmle.code.csharp.dataflow.internal.TaintTrackingPrivate
}
```
and get rid of `TaintTrackingUtil.qll`

Reply:
Done